src/Lean/Linter/Sets.lean

```diff
@@ -0,0 +1,34 @@
+/-
+Copyright (c) 2025 Lean FRO, LLC. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Anne Baanen
+-/
+prelude
+import Lean.Elab.Command
+import Lean.Linter.Basic
+
+namespace Lean.Linter
+
+/-- Add a new linter set that contains the given linters. -/
+def insertLinterSet [MonadEnv m] (setName : Name) (linterNames : Array Name) : m Unit :=
+  modifyEnv (linterSetsExt.addEntry · (setName, linterNames))
+
+/-- `registerSet` wraps `registerOption` by setting relevant values. -/
+def registerSet (setName : Name) (ref : Name := by exact decl_name%) : IO (Lean.Option Bool) := do
+  registerOption setName {
+    declName := ref
+    defValue := false
+    group := "linterSet"
+    descr := ""
```

Comment:
(I have no idea if this should also be populated.)

Reply:
I copied this from a few other `registerOption` calls, not sure what `descr` should be set to otherwise.